src/hipscat/catalog/partition_info.py

```diff
@@ -43,6 +43,15 @@ def get_highest_order(self) -> int:
 
         return highest_order
 
+    def write_to_file(self, partition_info_file: FilePointer):
+        """Write all partition data to CSV file.
+
+        Args:
+            partition_info_file: FilePointer to where the `partition_info.csv`
+            file will be written
```

Comment:
```suggestion
                file will be written
```

See https://hipscat--140.org.readthedocs.build/en/140/autoapi/hipscat/catalog/partition_info/index.html#hipscat.catalog.partition_info.PartitionInfo.write_to_file

Reply:
Thank you! I've fixed it.